test/test_linalg.py

```diff
@@ -1183,6 +1183,17 @@ def test_autograd_and_jit(self, device, dtype):
             ('norm', (S, S), ('fro', [0, 1]), 'fro_dim'),
             ('norm', (S, S), ('nuc',), 'nuc', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
             ('norm', (S, S), ('nuc', [0, 1]), 'nuc_dim', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+            ('norm', (S, S, S), (inf, ), 'flatten_3d_inf', (), NO_ARGS, [], {'flatten': True}),
```

Comment:
We should create an OpInfo for torch.linalg.norm. It doesn't have to be in this PR.

In a follow-up PR we should also investigate the backward performance of torch.linalg.norm vs torch.norm. Before deprecating torch.norm we should ensure it's comparable, which may mean implementing a custom backward for torch.linalg.norm.